### PR TITLE
fixed a few bugs.

### PR DESCRIPTION
**Pull request by NathanSalapat**
Some of these things may have been intentional, but they looked like bugs to me.
Changes:
Only bottoms of poles are visible in creative inventory.
Removed one crafting recipe for the right arrow, as the left arrow didn't have a similar recipe.
Made the signs attached nodes, so they drop when the node they were placed on is dug.
Made signs able to be dug.

Outstanding bug:
Digging tops of poles gives no drop. I couldn't figure that one out. An option would be to make that not diggable at all, so players have to dig the bottom node which then gives a drop as would be expected.

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -133,13 +133,6 @@ minetest.register_craft({
 })
 
 minetest.register_craft({
-	output = "hiking:"..colour.."_arrow_left",
```
Comment (csonto) [inline init.lua:136]: Copypasta. Thanks for spotting this.

File: init.lua [hunk 2]
```diff
@@ -26,7 +26,7 @@ local basic_properties = {
 		wall_bottom = {-0.4375, -0.5, -0.3125, 0.4375, -0.4375, 0.3125},
 		wall_side   = {-0.5, -0.3125, -0.4375, -0.4375, 0.3125, 0.4375},
 	},
-	groups = {snappy=1}, --2,dig_immediate=2,attached_node=1},
+	groups = {snappy=1, oddly_breakable_by_hand=2, attached_node=1},
```
Comment (csonto) [inline init.lua:29]: Hmmm, I could not find a good API documentation describing those little details. I think I now see what this is supposed to do. +1

**Comment by csonto** [comment]
Hmmm, I was happy enough to make poles work and be duggable. Still it is two objects, unlike beds. Could not make it into one... I think I should create issues for the known problems.

Thanks for help!

**Comment by NathanSalapat** [comment]
One option would be just to make the poles one node that has a height of
two nodes, then digging it at any point would remove the entire pole, drops
should also be fixed. This would allow somebody to place an object
intersecting in the upper half though.

**Pull request merged**